src/overview.md

```diff
@@ -19,16 +19,16 @@ we'll talk about that later.
 
 **TODO: someone else should confirm this vvv**
 
-- User writes a program and invokes `rustc` on it (possibly through `cargo`).
-- First, we parse command line flags, etc. This is done in [`librustc_driver`].
-  We now know what the exact work is we need to do (e.g. which nightly features
-  are enabled, whether we are doing a `check`-only build or emiting LLVM-IR or
-  a full compilation).
-- Then, we start to do compilation...
-- We first [_lex_ the user program][lex]. This turns the program into a stream
-  of _tokens_ (yes, the same sort of tokens as `proc_macros` (sort of)).
-  [`StringReader`] from [`librustc_parse`] integrates [`librustc_lexer`] with
-  `rustc` data structures.
+- The compile process begins when a user writes a Rust source program in text and invokes the `rustc` compiler on it. The work that the compiler needs to perform is defined with command line options. For example, it is possible to optionally enable nightly features, perform `check`-only builds, or emit LLVM-IR rather than complete the entire compile process defined here. The `rustc` executable call may be indirect through the use of `cargo`.
```

Comment:
Should we have a soft wrap here?

Reply:
Open question how we will format the entire document.  I am thinking that maybe this will become paragraph-based body copy rather than a bullet list?

Reply:
@mark-i-m Open issue here if we merge